src/pages/SongPage.tsx

```diff
@@ -59,6 +62,41 @@ const SongPage: React.FC = () => {
       console.error(e);
     }
   }
+
+  function PrevButton(songNumber: number) {
```

Comment:
nit: make function name start with lowercase letter, and be a verb, ie: `renderPrevButton()`

Reply:
Eh, I see in other places it starts with capital letter too. just adding the word `Render` at the front of it would be ok too